warehouse/integrations/snowflake/snowflake.go

```diff
@@ -843,11 +823,12 @@ func (sf *Snowflake) LoadIdentityMappingsTable(ctx context.Context) error {
 	return nil
 }
 
-// ShouldAppend returns true if:
+// ShouldMerge returns true if:
 // * the load table strategy is "append" mode
 // * the uploader says we can append
```

Comment:
Good catch, changed to:

```go
// ShouldMerge returns true if:
// * the server configuration says we can merge
// * the user opted-in
```

I removed the uploader from the comment since it's not being called here.